### PR TITLE
Return None when trying to parse a string as a float from SQL sources.

### LINKED ISSUE #106: Metadata update trying to parse empty string into float
```
2022-03-28 14:53:45,236 ERROR timeseer.cli MainProcess : could not convert string to float: ''
Traceback (most recent call last):
  File "timeseer\cli.py", line 105, in serve
  File "timeseer\cli_actions\validate_source.py", line 90, in validate_source
  File "venv\lib\site-packages\kukur\source\test.py", line 25, in search
  File "timeseer\source\app.py", line 95, in search_in_source
  File "venv\lib\site-packages\kukur\source\__init__.py", line 132, in search
  File "venv\lib\site-packages\kukur\source\sql.py", line 123, in search
  File "venv\lib\site-packages\kukur\source\sql.py", line 255, in __search_metadata
  File "venv\lib\site-packages\kukur\metadata\__init__.py", line 93, in coerce_field
  File "venv\lib\site-packages\kukur\metadata\fields.py", line 69, in deserialize
  File "venv\lib\site-packages\kukur\metadata\fields.py", line 88, in _parse_float
ValueError: could not convert string to float: ''
```

### PR DESCRIPTION
**Pull request by WouterWouters**
This closes #106

### CODE REVIEW

File: tests/source/test_odbc.py [hunk 1]
```diff
@@ -221,6 +221,36 @@ def test_accuracy_percentage_metadata() -> None:
...
+        );
+
+        insert into Metadata (series_name, accuracy_percentage, limit_low, limit_high) values ('random', 'two', 0, 10);
+        """
+    )
+    metadata = source.get_metadata(SeriesSelector("dummy", "random"))
+    assert metadata.get_field(fields.AccuracyPercentage) == None
+    assert metadata.get_field(fields.LimitLowPhysical) == 0
+    assert metadata.get_field(fields.LimitHighPhysical) == 10
+    assert metadata.get_field(fields.Accuracy) == None
```
Comment (jhoekx) [inline tests/source/test_odbc.py:251]: Always compare `None` with `is None`.

File: kukur/source/sql.py [hunk 2]
```diff
@@ -145,9 +145,12 @@ def get_metadata(self, selector: SeriesSelector) -> Metadata:
...
                 if isinstance(value, str) and value == "":
                     continue
-                metadata.coerce_field(
-                    name, self._metadata_value_mapper.from_source(name, value)
-                )
+                try:
+                    metadata.coerce_field(
+                        name, self._metadata_value_mapper.from_source(name, value)
+                    )
+                except ValueError:
```
Comment (jhoekx) [inline kukur/source/sql.py:152]: The main question for me was: why is the `ValueError` thrown?

The two lines before this should already check if it's an empty string.

**Review by jhoekx — CHANGES_REQUESTED** [review]

File: kukur/source/sql.py (hunk 2 above)
Comment (WouterWouters) [reply kukur/source/sql.py:152]: The `ValueError` is not thrown when it is an empty string, this is indeed checked. But when there is a real string value that cannot be parsed as a float, the value error is thrown.

Comment (WouterWouters) [reply kukur/source/sql.py:152]: This exact message is thrown when a string is not empty but holds one or more spaces.

**Pull request merged**